src/util_string.hpp

```diff
@@ -6,14 +6,65 @@
 namespace Sass {
 namespace Util {
 
-std::string rtrim(const std::string& str);
+std::string rtrim(std::string str);
 
 std::string normalize_newlines(const std::string& str);
 std::string normalize_underscores(const std::string& str);
 std::string normalize_decimals(const std::string& str);
 char opening_bracket_for(char closing_bracket);
 char closing_bracket_for(char opening_bracket);
 
+// ASCII routines based on Abseil:
+// https://github.com/abseil/abseil-cpp/blob/8f11724067248acc330b4d1f12f0c76d03f2cfb1/absl/strings/ascii.h
```

Comment:
Can Apache License 2.0 stuff be included in MIT projects. IMO Apache License has more restrictions?

Reply:
I'm not sure. Just to be on the safe side, I've re-implemented these functions myself.

My implementation is not as fast as the Abseil one (no lookup tables) but I don't think these are a bottleneck for us, and we can implement lookup tables ourselves later on.

Reply:
Thanks, better safe than sorry